modules/local/estimate_hb_perc.nf

```diff
@@ -0,0 +1,46 @@
+process ESTIMATE_HB_PERC {
+    tag "estimate_hb_perc"
```

Comment:
As this will be run several times if there is more than one sample, it would be good to have "$meta.id" as a tag